README.md

```diff
@@ -46,8 +46,8 @@ external Git source code hosting providers are available:
 
 ## Building
 
-[JDK 14](http://jdk.java.net/14/) or later and [Gradle](https://gradle.org/)
-6.6 or later is required for building. To build the project on macOS or
+[JDK 16](http://jdk.java.net/16/) or later and [Gradle](https://gradle.org/)
```

Comment:
How often is the minimum JDK version for Skara updated?  If it's always the last GA'd version of the JDK, perhaps that sort of phrasing should be used instead?

Reply:
Thanks for the explanation:  https://mail.openjdk.java.net/pipermail/skara-dev/2021-July/005097.html

I agree that the JDK version needs to remain explicit.